src/Functions/ChangeLog.md

```diff
@@ -18,8 +18,10 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-* Implemented Functions Stacks API which describes what runtimes and runtime versions are available for function app creation
-* Bug fixing
```

Comment:
Could you please update the release notes to the following?

```
Get-AzFunctionApp:
* [Breaking Change] Removed `IncludeSlot` switch parameter from all but one parameter set. The cmdlet now supports retrieving deployment slots in the results when `-IncludeSlot` is specified. 

New-AzFunctionApp:
* Fixed -DisableApplicationInsights so that no application insights project is created when this option is specified. [#12728]
* [Breaking Change] Removed support to create PowerShell 6.2 function apps.
* [Breaking Change] Changed the default runtime version in Functions version 3 on Windows for PowerShell function apps from 6.2 to 7.0 when the RuntimeVersion parameter is not specified.
* [Breaking Change] Changed the default runtime version in Functions version 3 on Windows and Linux for Node function apps from 10 to 12 when the RuntimeVersion parameter is not specified.
* [Breaking Change] Changed the default runtime version in Functions version 3 on Linux for Python function apps from 3.7 to 3.8 when the RuntimeVersion parameter is not specified.


```